packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -143,8 +143,8 @@ export interface GridOptions {
   editRowsModel?: GridEditRowsModel;
   /**
    * Filtering can be processed on the server or client-side.
-   * Set it to 'client' if you would like to handle filtering on the client-side.
```

Comment:
This still seem relevant, why remove it?

Reply:
The default is `client`, so I added `@default`. So I removed it  from here as I thought we need not explicitly set it to `client` because it is default.

Reply:
In the core, these comments are used in boolean props. There, the default value goes to `@default` and the comment only mentions what the non-default value does.

![image](https://user-images.githubusercontent.com/42154031/126221714-f0903801-02e5-4210-ba6b-35755f470d23.png)

I don't have any preference here, as long as we keep consistency: remove from all other feature mode props or keep this line.

Reply:
In this case, we can move slower: only add the default mention, it's already an improvement.

Reply:
I already have added the `@default`  tag.